tests/ha/clvm.pm

```diff
@@ -29,15 +29,15 @@ sub run() {
         type_string "echo wait until clvmd resource is created\n";
     }
     else {
-        assert_script_run q(EDITOR="sed -ie '$ a primitive clvmd ocf:lvm2:clvmd op monitor interval=60 timeout=60'" crm configure edit);    #create clvmd primitive
-        assert_script_run q(EDITOR="sed -ie 's/^\\(group base-group.*\\)/\1 clvmd/'" crm configure edit);                                   #add clvmd to base-group
-        sleep 10;                                                                                                                           #wait to get clvmd running on all nodes
+        assert_script_run q(EDITOR="sed -ie '$ a primitive clvmd ocf:heartbeat:clvm op monitor interval=60 timeout=60'" crm configure edit);    #create clvmd primitive
```

Comment:
It does exist in both. 

"crm configure primitive clvm ocf:heartbeat:clvm" works.

However now it is simplified to "primitive clvm clvm".
That might be enough.